code-challenges.js

```diff
@@ -9,67 +9,156 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
-var temp1 = 85
-var temp2 = 350
-var temp3 = 212
-
+// var temp1 = 85
+// var temp2 = 350
+// var temp3 = 212
 
+// const temperatureOutcome = (number) => {
+//       if (number < 212) {
+//         return `${number} is below boiling point`
+//     } else if (number === 212) {
+//         return `${number} is at boiling point`
+//     } else if (number > 212) {
+//         return `${number} is above boiling point`
+//     }
```

Comment:
this is great! all the logic and indentation is there! What if someone calls this function with a string or non-number argument?